[www] Blog Adding "Why Narative loves Gatsby" post

brotzky opened the pull request:
Hi!

I've shared this blog post with @KyleAMathews through twitter and he asked if we'd like to add it to the official Gatsby blog. So, I'd like to add a blog post we've been working on recently Narative, "Why Narative loves Gatsby".

If possible, this would be amazing to have live by June 18th. Otherwise, any time that fits your schedule at Gatsby!

Let me know if I missed anything. First PR to Gatsby!

We're excited to be using your software :). If there's any way we can lend our skills to help improve it please let us know! For now, we'd like to share some kind words through this blog post.

✌️[Narative](https://narative.co)

m-allanson commented:
Thanks @brotzky! Preview url: https://deploy-preview-5883--gatsbyjs.netlify.com/blog/2018-06-18-why-narative-loves-gatsby/

It looks like there's a space in the filename of the plugins image, which is preventing it from loading.

m-allanson commented on docs/blog/2018-06-18-why-narative-loves-gatsby/index.md:
```diff
@@ -0,0 +1,58 @@
+
+Gatsby allows our team to use the powerful React API when creating page based websites with almost no drawbacks. All the principles and techniques we've learned from using React for the last few years can be leveraged with Gatsby. This has enabled us to create a team that is extremely dynamic and cohesive without the worry of creating silos or losing knowledge across teams. Our team is able to le ...
+
+One concern we had when adopting Gatsby is that page based websites are not the same web apps. Even though we're using similar tools to build them there are different principles that drive each type of development. Luckily, our team doesn't have to worry about that problem, because the team behind Gatsby is focused on delivering the perfect experience to end users for us. The Gatsby team has bake ...
```
missing _as_? 

> are not the same _as_ web apps

m-allanson commented on docs/blog/2018-06-18-why-narative-loves-gatsby/index.md:
```diff
@@ -0,0 +1,58 @@
+
+Narative strives for technical excellence for all its products, which can be achieved with React, Gatsby, and GraphQL for us. We've landed on a solution that allows our team to stay cohesive and flexible. Thanks to the popularization of React and the principles behind it Narative has been able to build page based websites and web apps with the same expertise. In the past, it was hard to share kno ...
+
+Gatsby streamlines our process with an excellent developer experience and all the features you could ask for in 2018. The heavy lifting of delivering a technically sound website with performance that users will love has slowly shifted towards frameworks such as Gatsby, which has freed up our team to focus on even better best experiences and products.
```
remove _best_?

brotzky commented:
@m-allanson thanks so much! great suggestions. pushed up a commit to resolve those.
- added the word as
- removed the word best
- removed leading space in filename

jlengstorf commented:
Hiya @brotzky! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs
